ext/objspace/lib/objspace.rb

```diff
@@ -11,13 +11,14 @@ class << self
 
   module_function
 
-  # call-seq:
-  #   ObjectSpace.dump(obj[, output: :string]) -> "{ ... }"
-  #   ObjectSpace.dump(obj, output: :file) -> #<File:/tmp/rubyobj20131125-88733-1xkfmpv.json>
-  #   ObjectSpace.dump(obj, output: :stdout) -> nil
-  #
   # Dump the contents of a ruby object as JSON.
   #
+  # _output_ can be one of: +:stdout+, +:file+, +:string+, or IO object.
+  #
+  # * +:file+ means dumping to a tempfile and returning corresponding File object;
+  # * +:stdout+ means printing the dump and returning +nil+;
+  # * +:string+ means returning a string with the dump.
```

Comment:
For consistency, we should state that IO object outputs to the provided object and returns the provided object.  The output part is probably obvious, but the return value may not be.

Reply:
This is fixed.